src/screens/Departures/components/EstimatedCallItem.tsx

```diff
@@ -54,13 +55,15 @@ export default function EstimatedCallItem({
 
   return (
     <TouchableOpacity
-      onPress={() =>
-        navigateToDetails(
-          departure.serviceJourney?.id,
-          departure.expectedDepartureTime,
-          departure.quay?.id,
-        )
-      }
+      onPress={() => {
+        if (departure.serviceJourney)
```

Comment:
Burde dette vært knapp og hatt trykk-animasjon hvis det har ingen effekt å trykke på den?

Reply:
vi burde kanskje visuelt si om den kan trykkes på eller ikke. 
I utgangspunktet er denne checken her for å unngå å generere en feil, og det skal heller ikke kunne skje, for du har i praksis aldri estimated calls uten at du har en serviceJourney.